README.md

```diff
@@ -1,38 +1,103 @@
 # SimpleIndexer
 
-`SimpleIndexer` use `DocumentArrayMemmap` for indexing `Document`. It is recommended to be used in most of the simple use cases when you have less than one million `Document`. 
-
-`SimpleIndexer` leverages `DocumentArrayMmap`'s [`match`](https://docs.jina.ai/api/jina.types.arrays.abstract/#jina.types.arrays.abstract.AbstractDocumentArray.match) function and searches the `k` nearst neighbors for the query `Document` based on their `embedding` field by the brutal-force search. By default, it calculates the `cosine` distance and returns all the indexed `Document`.
+`SimpleIndexer` uses `DocumentArrayMemmap` for indexing `Document`. It is recommended to be used in most of the simple use cases when you have less than one million `Document`. 
 
+`SimpleIndexer` leverages `DocumentArrayMmap`'s [`match`](https://docs.jina.ai/api/jina.types.arrays.mixins.match/?module-jina.types.arrays.mixins.match) function and searches the `k` nearest neighbors for the query `Document` based on their `embedding` field by the brutal-force search. By default, it calculates the `cosine` distance and returns all the indexed `Document`.
 
 
 ## Advanced Usages
 
 ### Configure the index directory
 
 `SimpleIndexer` stores the `Document` at the directory, which is specified by `workspace` field under the [`metas`](https://docs.jina.ai/fundamentals/executor/executor-built-in-features/#meta-attributes) attribute. 
+You can override the default configuration as below,
+
+```python
+f = Flow().add(
+    uses='jinahub://SimpleIndexer',
+    uses_metas={'workspace': '/my/tmp_folder'})
+```
+
+Find more information about how to override `metas` attributes at [Jina Docs](https://docs.jina.ai/fundamentals/flow/add-exec-to-flow/#override-metas-configuration)
+
+
+### Choose embeddings
+The [recursive structures](https://docs.jina.ai/fundamentals/document/document-api/#recursive-nested-document) of Documents can be quite useful to represent the Documents at different semantic granularity. 
```

Comment:
Maybe we should move these guideliens to the main docs, in a page titled something like 'Best Practices with Indexers'. These suggestions are pretty general and not just about SimpleIndexer

Reply:
Yes, I also find the explanation of `traversal_paths` is missing from the Jina docs.